util/src/dbutil.cpp

```diff
@@ -144,7 +154,7 @@ bool checkPsqlDatbase(
   std::string database = connStr_.substr(0, colonPos);
   std::string options  = connStr_.substr(colonPos + 1);
 
-  std::vector<std::string> odbOpts = createOdbOptions(options);
+  const std::vector<std::string>& odbOpts = *createOdbOptions(options);
```

Comment:
Putting a reference on a temporary object results in a segfault.
Introduced bug was fixed in #320.